pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -76,7 +76,7 @@ start_test "Test that quit does not emit unwanted logging output"
 # Unwanted: between the point the command starts until it
 # either prints the final ok message or fails with some error
 # (e.g. due to no definite answer from the server).
-send "echo marker; $argv quit 2>&1 | grep '^\\(\[IWEF\]\[0-9\]\\)' \r"
+send "echo marker; $argv quit 2>&1 | grep -vE '^\[IWEF\]\[0-9\]+ grpc: '\r"
```

Comment:
Not good enough, I think! We need to match

> W171020 12:24:14.884065 21 vendor/google.golang.org/grpc/clientconn.go:1034  grpc: addrConn.resetTransport failed to create client transport:

So I'm missing a `.*` somewhere.

Reply:
Good point! Fixed in #19402.